### PR TITLE
Add find command to dev guide

### LINKED ISSUE #98: Add Find feature to Developer Guide

### PR DESCRIPTION
**Pull request by kevinchangjk**
**Resolves #98**

Note that at this point, the find function hasn't actually been implemented yet. Will add in the UML sequence diagram along with additional details (errors, cases) when the command is done.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -239,22 +239,41 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+
+Step 3. The `execute` method of `DeleteCommand` is then called on the object, which returns a `CommandResult` object.
+
+Step 4. This finds the `person` from the list from `model#getFilteredPersonList` by its employee ID which is `1` in this case. If there does not exist a `person` object with employee ID of `1`, a `CommandException` will be thrown and a message indicating invalid ID given will be shown. If the `person` object exists, then using `model#deletePerson()`, the `person` object is deleted from the `database`.
+
+Step 5. storage#saveDatabase is then called on the current `database`, updates the database to not contain the deleted `person`.
+
+### Find feature
+
+This section explains the implementation of the `find` feature. The command takes in a number of parameters, which serve as the "filters" for the finding/searching function. At present, we have implemented finding by name, department, position, and any combination of these three. Thus it is possible to use these altogether to search for a person with high specificity.
```
Comment (Kok-je) [inline docs/DeveloperGuide.md:262]: The description can be more comprehensive here.

**Review by Kok-je — COMMENTED** [review]
Excellent punctuation and formatting. Keep up the good work Kevin!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (prit3010) [inline docs/DeveloperGuide.md:262]: A clear description of the feature

**Review by prit3010 — APPROVED** [review]
LTGM

**Pull request merged**